src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * To perform operations on the list of tasks
+ */
+public class TaskList {
+    public static ArrayList<Task> list = new ArrayList<>();
+    static int count = 0;
```

Comment:
Perhaps, you can consider using the `.size()` method from the ArrayList class. This way, you would not need to manually keep track of the number of tasks in the task list.